mediacapture-streams/obtaining-local-multimedia-content/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
```

Comment:
@dontcallmedom can you defend this practice, please? I regard our chapter structure as reasonably irrelevant to good test grouping.